src/main/java/com/glitchcog/fontificator/bot/Message.java

```diff
@@ -414,42 +414,25 @@ private boolean isConfigChanged(ConfigMessage messageConfig, ConfigEmoji emojiCo
 
             // Bank to pull FrankerFaceZ badges from
             TypedEmojiMap ffzBadgeBank = emojiManager.getEmojiByType(EmojiType.FRANKERFACEZ_BADGE);
+            Map<String, LazyLoadEmoji> ffzBadgeMap = new LinkedHashMap<String, LazyLoadEmoji>();
```

Comment:
I originally used a LinkedHashMap to collect all the badges to be displayed, both Twitch and FFZ, because I could control the order they were added.
It doesn't look like this ffzBadgeMap variable needs to be ordered, since it only will contain at most the FFZ mod override and the FFZ donor badges, but in principle I think it should be, since whatever is in this map will get drawn in some order as badges. In theory, FFZ might add five more badges tomorrow, and then we'd have to change it back.